homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,27 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int a=0;
+    int b=1;
+
+    if(sequence==0){
+        return 0;
+    }
+    for (int i =0; i<sequence-1;i++){
+
+        b+=a;
+        a=b-a;
+    }
+    return b;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence ==0){
+        return 0;
+    }
+    if(sequence ==1){
+        return 1;
+    }
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`